docs/docs/store.mdx

```diff
@@ -37,48 +37,134 @@ When you want to update the store, you can call the `update()` method passing th
 
 ```ts title="session.service.ts"
 import { SessionStore } from './session.store';
- 
+
 export class SessionService {
   constructor(private sessionStore: SessionStore) {}
 
   updateUserName(newName: string) {
     this.sessionStore.update({ name: newName });
-  }  
+  }
 }
 ```
 
 The second `update()` option gives you more control. It receives a `callback` function, which gets the current state, and returns a new **immutable** state, which will be the new value of the store. For example:
 
 ```ts title="session.service.ts"
 import { SessionStore } from './session.store';
- 
+
 export class SessionService {
   constructor(private sessionStore: SessionStore) {}
 
   updateUserName(newName: string) {
     this.sessionStore.update(state => ({
       name: newName
     }));
-  }  
+  }
+}
+```
+
+### `set()`
+
+When you don't just want to update the current store state but replace the state completely, you can call the `set()` method passing the new `state`:
+
+```ts title="session.service.ts"
+import { SessionStore } from './session.store';
```

Comment:
We can' use the name `set` because it's an EntityStore method.

Reply:
A few alternatives ideas:

`put`

The positives would be it's familiar terminology to people, it performs as expected. With the caveat that in the HTTP spec it's idempotent and this method would not be technically idempotent if a akitaPre hook performs a side effect.

`overwrite`

At a glance it feels descriptive of the functionality it performs. When paired with `update()` give an immediate intuition about the differences between them. Does not feel as familiar in the world of CRUD updating.

`setState`

Very explicit about what it does. Is close to the @internal `_setState()` method which could cause confusion. Inclusion of the word 'State' doesn't gel as well with the update(), sounds less like an alternative and more like different functionality.

My personal preference order: `overwrite` -> `put` -> `setState`.

Reply:
let's go with overwrite

Reply:
Done.